cmd/mailchain/internal/http/handlers/openapi.go

```diff
@@ -133,7 +133,7 @@ func spec() string {
         }
       },
       "post": {
-        "description": "Securely send message on the protocol and network specified in the query string to the address.\nOnly the private key holder for the recipient address can decrypted any encrypted contents.\n\nCreate mailchain message\nEncrypt content with public key\nStore message\nEncrypt location\nStore encrypted location on the blockchain.",
+        "description": "Securely send message on the protocol and network specified in the query string to the address.\nOnly the private key holder for the recipient address can decrypt any encrypted contents.\n\nCreate mailchain message\nEncrypt content with public key\nStore message\nEncrypt location\nStore encrypted location on the blockchain.",
```

Comment:
Not sure if this file and specs.json needs to be updated. Looking at the makefile, it looks like these will be generated at run-time?

Can we then delete these 2 files from the repo?

Reply:
`cmd/mailchain/internal/http/handlers/send_message.go` is used to generate `docs/openapi/spec.json` then `cmd/mailchain/internal/http/handlers/openapi.go`. Its a multi step process but you raise a good point in that `docs/openapi/spec.json` has no use after it has been generated by the application. I left it in the repo for reference online. Its something we can revisit later. 

You would not need to make the same changes in multiple places, instead you can change`cmd/mailchain/internal/http/handlers/send_message.go` and run `make openapi` and it will recreate the other two files.